syntax = "proto3";

package pressure;

message Input {
    int32 fileNumber = 1;  // 文件编号
    repeated float pressureData = 2; // 压力数据
}

message Output {
    repeated float pressureData = 1; // 压力数据
    bool hasDetectedPressure = 2; // 检测结果
    bool hasThreat = 3;
}